deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -592,6 +592,22 @@
                 (assoc :block/updated-at updated-at))]
     (dissoc block :title :body :anchor)))
 
+(defn fix-duplicate-id
```

Comment:
Optional: Could be useful to `log/info` when we re-write/fix id so that users always can always inspect modified content

Reply:
Good point, updated.